composer.json

```diff
@@ -3,10 +3,21 @@
         {
             "name": "Tom",
             "email": "tomjharding@live.co.uk"
+        },
+        {
+            "name": "Woody Gilk",
+            "email": "woody.gilk@gmail.com",
+            "role": "Contributor"
         }
     ],
     "autoload": {
-        "files": ["src/Combinators.php"]
+        "psr-4": {
+            "PhpFp\\Combinators\\": "src/"
+        },
+        "files": [
+            "src/functions.php",
+            "src/curried.php"
```

Comment:
Should both be exposed? The "uncurried" ones aren't all totally uncurried (thinking of `compose`, in particular), so it might be more helpful to hide the "original" functions and move them out of sight and mind.